projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -366,7 +366,7 @@ interface ContextToggleItem extends ToggleItem<ExplorerContextScope> {
   value: ExplorerContextScope;
 }
 
-interface InitialExplorerState {
+export interface InitialExplorerState {
```

Comment:
@palbizu is this needed?

Reply:
Yes, should be needed if we want to reference the interface from outside.